api-specifications/properties.json

```diff
@@ -0,0 +1,625 @@
+{
+  "scriptappy": "1.0.0",
+  "info": {
+    "name": "@nebula.js/sn-action-button:properties",
+    "description": "Table generic object definition",
+    "version": "1.14.3",
+    "license": "MIT",
+    "stability": "experimental",
```

Comment:
Is this still experimental?

Reply:
Not sure, when does it stop being experimental?